clang/test/SemaHLSL/resource_binding_attr_error_udt.hlsl

```diff
@@ -115,12 +114,14 @@ struct Eg13{
   MySRV s1;
   MySRV s2;
 };
-// expected-warning@+4{{binding type 'u' only applies to types containing UAV resources}}
 // expected-warning@+3{{binding type 'u' only applies to types containing UAV resources}}
-// expected-warning@+2{{binding type 'u' only applies to types containing UAV resources}}
+// expected-error@+2{{binding type 'u' cannot be applied more than once}}

```

Comment:
This extra diagnostic relates to my earlier comment, preventing multiple copies of the same diagnostic.

Reply:
Since there are 2 extra 'u' bindings we should be reporting 2 errors.